src/webpub_manifest_parser/epub/ast.py

```diff
@@ -0,0 +1,51 @@
+from webpub_manifest_parser.core.ast import LinkProperties, Metadata, Node
+from webpub_manifest_parser.core.parsers import ArrayParser, EnumParser
+from webpub_manifest_parser.core.properties import (
+    EnumProperty,
+    IntegerProperty,
+    Property,
+    StringProperty,
+    TypeProperty,
+    URIProperty,
+)
+
+
+class EPUBPresentationHints(Node):
+    """EPUB presentation hints."""
+
+    layout = EnumProperty("layout", required=False, items=["fixed", "reflowable"])
+
+
+class EPUBMetadata(Metadata):
+    """EPUB metadata."""
+
+    presentation = TypeProperty(
+        "presentation", required=False, nested_type=EPUBPresentationHints
+    )
+
+
+class EPUBEncryptionSettings(Node):
```

Comment:
This will soonish be extracted from the EPUB extension into a DRM module instead. This was agreed on but not yet updated in the spec. So you can probably rename it to `DRMEncrytionSettings` or `EncryptionSettings` (on mobile we called it `Encryption`).

Reply:
Since there other classes in `epub` module, do you think it makes sense to wait until you update the spec and then update the library accordingly? Because I think it would be better to extract it to a separate module

Reply:
The encryption settings are needed for RWPM profiles protected with LCP as well, which are not necessary EPUBs. So as long as the `EncryptionSettings` object is accessible outside of the EPUB scope (`EPUBManifest`?), this issue is not pressing.

Here's an example for audiobooks: https://readium.org/lcp-specs/notes/lcp-for-audiobooks.html